### ISSUE TITLE
Your website and apis are gone

### ISSUE THREAD

**Issue by shreenologyinfotech** [rest7/api#22]
I have seen the your website and  Apis are stopped  I am  not able to see anything on your website.

**Comment by flyingwebmonkey** [comment]
Any info on this? Is it possible for me to get the api for vector raster app to host ive been building a project based on this api fir over a year and I am needing the api to complete it. Thanks

**Comment by shreenologyinfotech** [comment]
Yes  I have  made  my own solution For that And  Developed with few  functions and  Now  I am using this in my project. Can we discuss this  on call or on mail of me shreenologyinfotech@gmail.com. May be I can help you out.
